core/src/km_core_keyboard_api.cpp

```diff
@@ -40,16 +40,18 @@ namespace
 }  // namespace
 
 km_core_status
-keyboard_load_from_blob_internal(
+km_core_keyboard_load_from_blob(
```

Comment:
Hmm, removing `keyboard_load_from_blob_internal` means that we'll have to initialize a `std::vector` twice when called from WASM. Don't know how much overhead that has.

Reply:
well— could _retain_ `keyboard_load_from_blob_internal` passing in the vector?

Reply:
Currently, we don't have the load from blob code in epic/web-core, is that right? Happy for a refactor back if that makes sense, but as it isn't landing in 18.0, I thought we could cleanup the source here at this point, given the km_core_keyboard_load_from_blob was literally a wrap-in-vector and call internal two liner.